mesh/profiles/views.py

```diff
@@ -1,22 +1,49 @@
 from django.core.exceptions import ObjectDoesNotExist
-from django.views.decorators.csrf import csrf_exempt
 from django.http import JsonResponse
+from django.core import serializers
+from mesh.accounts.models import Account
 
 from mesh.profiles.models import Profile
 
+from django.views import View
 
-def bio_view(request):
-    if request.method == "POST":
-        data = request.POST
-        if 'accountID' not in data:
-            return JsonResponse({'error': 'Invalid request. Missing accountID field.'}, status=401)
-        if 'biography' not in data:
-            return JsonResponse({'error': 'Invalid request. Missing biography field.'}, status=400)
+class BiographyView(View):
+    """
+    View for getting an biography by accountId or updating an biography by accountID.
+    """
+    def get(self, request, account_id, *args, **kwargs):
+        """
+        Handle GET requests.
+
+        Retrieves a biography of the listed profile in JSON format.
+
+        Returns a JSON response containing biography of specified profile through id.
+        """
+        try:     
+            if(account_id is None):
+                return JsonResponse({'error': 'Invalid input. Missing account ID'}, status=401)
```

Comment:
Throw an exception instead of doing error handling with a jspn response and *sometimes* using exceptions. 

Exceptions are for error handling.

Reply:
Wouldn't it actually be better to use the `validate_json_and_required_fields` from the utils folder here and just catch the InvalidJsonFormat/MissingRequiredFields exceptions raised from that function, rather than raising an exception himself?

Reply:
Yea it would @JRSiwiecki

Reply:
Actually I just realized this isn't even json it's from the URL so that method would not work here, my bad 👎🏼